BUILD.md

```diff
@@ -150,5 +150,5 @@ More details: https://github.com/apache/cordova-plugin-whitelist
 To have logs dump in the terminal you can use ```--consolelogs``` options
 
 ```
-npm run ios -- --consolelogs
```

Comment:
I think this is correct. We are running that through npm script so the extra `--` will apply the next options to `cordova` in this case.